libs/ui/paginator/src/lib/paginator/paginator.component.ts

```diff
@@ -385,11 +399,13 @@ export class TsPaginatorComponent implements OnChanges, AfterViewInit {
    * @param destinationPage - The selected page number
    * @param currentPage - The current page number
    * @param pages - The collection of pages
+   * @param action - The action user takes, click previous page or next page
    */
   public changePage(
     destinationPage: number,
     currentPage: number,
     pages: TsPaginatorMenuItem[],
+    action = '',
```

Comment:
It seems like we could figure this out without adding another param? 

Something along the lines of
```
const direction = ((destinationPage - currentPage) > 0) ? 'next' : 'prev';
```

Reply:
Good point! Didn't look at existing params.